torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -165,7 +168,9 @@ def _retrieve_or_adapt_input_to_graph_set(
         # onnx-script auto wraps python number with op.Constants,
         # so we don't need to specifically process them.
         with onnxscript.evaluator.default_as(tracer):
-            output = onnxscript.opset18.Concat(*sequence_mixed_elements, axis=0)  # type: ignore[type-var]
+            output = onnxscript_apis.torchlib_opset().Concat(
```

Comment:
Ops in _building.py also needs the changes?

Reply:
The opset version is used in https://github.com/pytorch/pytorch/pull/141413/files#diff-c2f19ccd7bc57c1edd3281f4d758f616c933599307fded5d3ad3d063a5313222R115 which is then used in https://github.com/pytorch/pytorch/blob/4c1f50af5fa4835d3fa90063e7903a05e5a6df13/torch/onnx/_internal/exporter/_core.py#L691 and https://github.com/pytorch/pytorch/blob/4c1f50af5fa4835d3fa90063e7903a05e5a6df13/torch/onnx/_internal/exporter/_core.py#L445. So we should be good